docs/source/changelog.rst

```diff
@@ -26,9 +26,13 @@ We are pleased to announce the latest LiberTEM release, with many
 improvements since 0.5. We would like to highlight the contributions of our
 GSoc 2020 students `@AnandBaburajan <https://github.com/AnandBaburajan>`_ and
 `@twentyse7en <https://github.com/twentyse7en>`_, who implemented significant
-improvements in the areas of I/O and the user interface. Another highlight of
-this release is the support of NVidia GPUs, both via CuPy and via native
-libraries. A lot of work was done to implement tiled reading, resulting in a
+improvements in the areas of I/O and the user interface.
+
+Another highlight of this release is experimental support of NVidia GPUs, both
+via CuPy and via native libraries. The API is ready to be used, including support
+in the GUI. Performance optimization is still to be done (:issue:`946`).
```

Comment:
GPU support is activated for all mask-based analyses (virtual detector and Radial Fourier) for testing purposes, but will not bring a noticeable improvement of performance yet. GPU-based processing did show significant benefits for computationally heavy applications like the SSB implementation in https://github.com/Ptychography-4-0/ptychography.